lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -112,9 +118,14 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
     }
 
     markAsRead() {
-        this.notifications = [];
+        this.unreadNotifications = [];
+        this.notifications.forEach((notification: NotificationModel) => {
+            notification['read'] = true;
+        });
+
+        this.badgeHidden = !this.badgeVisibility();
+        this.storageService.setItem('notification-history', JSON.stringify(this.notifications));
```

Comment:
Please reuse NOTIFICATION_STORAGE field as it already contains this string

Reply:
reused NOTIFICATION_STORAGE